geomesa-compute/src/main/scala/org/locationtech/geomesa/compute/spark/GeoMesaSpark.scala

```diff
@@ -33,7 +32,8 @@ import org.geotools.data.{DataStore, Query}
 import org.geotools.factory.CommonFactoryFinder
 import org.locationtech.geomesa.core.data._
 import org.locationtech.geomesa.core.index.{IndexSchema, STIdxStrategy}
-import org.locationtech.geomesa.feature.{AvroFeatureDecoder, AvroFeatureEncoder, SimpleFeatureEncoder, AvroSimpleFeature}
+import org.locationtech.geomesa.feature._
+import org.locationtech.geomesa.feature.kryo.{SimpleFeatureSerializer, KryoFeatureSerializer}
```

Comment:
there is not actually a space there, im not sure why github shows one, it has the line numbers right